### ISSUE TITLE
Make `ockam tcp-inlet show` (no args) interactive by asking the user to choose from a list of tcp-inlets to show (tuify)

### ISSUE THREAD

**Issue by nazmulidris** [build-trust/ockam#6478]
# Current behavior

When a user runs `ockam tcp-inlet show` without any arguments, currently they are shown an error message & help text. Here's a screenshot of this.

![Image](https://github.com/build-trust/ockam/assets/2966499/6722a664-22fa-4036-a9b2-aeed7ee98abc)

# Desired behavior

Change the behavior of this command so that it becomes interactive with the user when no arguments are passed. Note that this can only be allowed to happen if:

- the terminal is interactive, and,
- the [`--no-input`](https://github.com/build-trust/ockam/blob/develop/implementations/rust/ockam/ockam_command/src/terminal/mod.rs#L265) or `--quiet` flags have *not* been passed, and,
- no arguments have been passed in to the command.

This what the user flow should look like in interactive mode.

1) Use the [tuify](https://crates.io/crates/r3bl_tuify) crate's `select_from_list()` function to ask the user to select from a list of tcp-inlet aliases, at the default node. They should be able to make multiple selections. If there are none, then simply display a message saying that none exist (not an error) and exit.

2) Once the user has made their selection(s), then run show for each selected tcp-inlets, and display the results of the `show` command for each tcp-inlets that they've selected.
   - They should be able to make a single selection.
   - If they have made no selections then exit by displaying a message to the user letting them know they did not make any selection.

> The following is a video showing a generic user flow that shows the user experience when a user is asked to choose a from a list of "things". Then an action is performed on each "thing". Finally status information is reported for each action that is performed on each "thing". Please note that this flow is *not* specific to this issue. 
> ![Image](https://github.com/build-trust/ockam/assets/2966499/19efea08-4bad-433a-bf8d-b1e709f69a15)
>  https://asciinema.org/a/612547

# Implementation details

- Please use the functions `select_multiple`, and `confirm_interactively` in this [file](https://github.com/build-trust/ockam/blob/develop/implementations/rust/ockam/ockam_command/src/terminal/mod.rs#L281).

- Here's a similar PR to get some ideas from: https://github.com/build-trust/ockam/pull/6480

- You can learn more about how to use the `select_from_list()` in the tuify crate [here](https://docs.rs/r3bl_tuify/latest/r3bl_tuify/). Here is an [example](https://github.com/r3bl-org/r3bl_rs_utils/blob/ockam-demo/tuify/examples/main_interactive.rs) that is very similar to the steps you might have to take for this issue.
- Here's a [file](https://github.com/build-trust/ockam/blob/develop/implementations/rust/ockam/ockam_command/src/terminal/mod.rs#L255) you can look at to detect when a terminal is interactive or not.

- Here's a [file](https://github.com/build-trust/ockam/blob/develop/implementations/rust/ockam/ockam_command/src/tcp/inlet/show.rs#L42) you can look at to see where the `ockam tcp-inlet show` command is implemented.

---
<i>

We love helping new contributors! ❤️
If you have questions or need help as you explore, please join us on [Discord](https://discord.gg/bewvnm6zqS). If you're looking for other issues to contribute to, please checkout our [good first issues](https://github.com/build-trust/ockam/labels/good%20first%20issue).
</i>

**Comment by makeittotop** [comment]
@nazmulidris  The description looks very thorough, thanks. Can I take it up?

**Comment by nazmulidris** [comment]
@makeittotop That's awesome, this is all yours. Please let us know if you have any questions as you explore. You can also ask questions on the contributors discord https://discord.gg/RAbjRr3kds

**Comment by makeittotop** [comment]
Thanks @nazmulidris . I'll start working on it very soon as I'm very busy at work for the next few days. Thanks for the patience.

**Comment by nazmulidris** [comment]
@makeittotop No worries. If you don't have time, feel free to unassign yourself from this issue. You can always come back later and request this issue to work on or some other one 👍🏽

**Comment by makeittotop** [comment]
No, I've time @nazmulidris . I'll submit the PR very soon

**Comment by nazmulidris** [comment]
@makeittotop Excellent 👍🏽 Looking forward to it

**Comment by makeittotop** [comment]
@nazmulidris reg: `the --no-output or --quiet flags have not been passed, and,...`  I couldn't find --no-output flag in the ockam command global options list. Did you mean --no-input instead?

**Comment by nazmulidris** [comment]
@makeittotop You are right it is `--no-input` (https://github.com/build-trust/ockam/blob/develop/implementations/rust/ockam/ockam_command/src/terminal/mod.rs#L265). Sorry about that, I will update the other issues to reflex this fix. 👍🏽

**Comment by nazmulidris** [comment]
@makeittotop You can also take a look at this PR as an example https://github.com/build-trust/ockam/pull/6480

- Please use the functions `select_multiple`, and `confirm_interactively` in this [file](https://github.com/build-trust/ockam/blob/develop/implementations/rust/ockam/ockam_command/src/terminal/mod.rs#L281).

**Comment by nazmulidris** [comment]
@makeittotop We made a minor UX (user experience) update to the issue today. Instead of asking a user for confirmation once they've made their selection, we can just skip that and allow them to perform the action. Since the action is not destructive, it results in a better UX for the user.

**Comment by makeittotop** [comment]
Thanks for the info. I'll make the necessary adjustment, and do the PR by
the end of the day
